src/client/ui/panels/HierarchyPanelContainer.js

```diff
@@ -135,16 +136,20 @@ class HierarchyPanelContainer extends Component {
   };
 
   renderNode = node => {
+    const isConflict = node.object.userData._missing || node.object.userData._duplicate;
+    const isMissingChild = node.object.userData._missing && !node.object.userData._isMissingRoot;
+    const isDuplicateChild = node.object.userData._duplicate && !node.object.userData._isDuplicateRoot;
     return (
       <div
         className={classNames("node", {
           "is-active": this.props.editor.selected && node.object.id === this.props.editor.selected.id,
-          conflict: node.object.userData._missing,
+          conflict: isConflict,
           "error-root": node.object.userData._isMissingRoot ? node.object.userData._missing : false,
-          disabled: node.object.userData._missing && !node.object.userData._isMissingRoot
+          "warning-root": node.object.userData._isDuplicateRoot ? node.object.userData._duplicate : false,
+          disabled: isMissingChild || isDuplicateChild
         })}
-        onMouseUp={node.object.userData._missing ? null : e => this.onMouseUpNode(e, node)}
-        onMouseDown={node.object.userData._missing ? e => e.stopPropagation() : null}
+        onMouseUp={isConflict ? null : e => this.onMouseUpNode(e, node)}
+        onMouseDown={isConflict ? e => e.stopPropagation() : null}
```

Comment:
Do we want to disable interaction with duplicate objects?

Reply:
I'm not sure about it, so I disable them at first. My concern is that if users move children to other duplicate roots and then save the file. The `.scene` file does not really take care of this situation since we don't modify the node name in the gltf file, which might made users confused when they reopen the `.scene` file and see all children are still under the first duplicate root.

Reply:
Fair enough. Please make a new issue for it so that we can deal with it later.